utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt

```diff
@@ -69,8 +70,18 @@ class UrlImageParser private constructor(
       val drawable = drawableFactory(resource)
       htmlContentTextView.post {
         htmlContentTextView.width {
-          val drawableHeight = drawable.intrinsicHeight
-          val drawableWidth = drawable.intrinsicWidth
+          var drawableHeight = drawable.intrinsicHeight
+          var drawableWidth = drawable.intrinsicWidth
+          val minimumImageSize = context.resources.getDimensionPixelSize(R.dimen.minimum_image_size)
+          if (drawableHeight <= minimumImageSize || drawableWidth <= minimumImageSize) {
```

Comment:
Please add kdoc for this calculation it will be nice.

Reply:
Added documentation PTAL

Reply:
Thanks @rt4914.